crates/e2e/tests/e2e/protocol_fee.rs

```diff
@@ -87,7 +91,7 @@ async fn surplus_fee_sell_order_test(web3: Web3) {
     // 1480603400674076736) = 1461589542731026166 DAI
     execute_test(
         web3.clone(),
-        fee_policy,
+        vec![protocol_fee],
```

Comment:
I think we can probably reduce this e2e test now to a single test that matches a limit and a swap order with different fee policies.

Reply:
Makes sense. I would refactor this in a separate PR. Probably, in scope of the https://github.com/cowprotocol/services/issues/2468